packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -159,8 +136,9 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     if (prevState.stickyTopHeight !== this.state.stickyTopHeight || prevState.stickyBottomHeight !== this.state.stickyBottomHeight) {
       this.notifySubscribers();
     }
-
-    this._async.setTimeout(this._onWindowResize, 0);
+    if (this._mutationObserver || this._stickies.size) {
```

Comment:
A comment on what's up here would be nice. Calling `_onWindowResize` on every component update seems wrong.

Reply:
This specific line change `this._async.setTimeout(this._onWindowResize, 0);` was not done by me. I tried finding it's significance but still not sure about this specific line here. I've only introduced a condition here, which makes sure `this._async.setTimeout(this._onWindowResize, 0);` is called only if there is at least one Sticky. `this._async.setTimeout(this._onWindowResize, 0);` has no significance if there is no Sticky.